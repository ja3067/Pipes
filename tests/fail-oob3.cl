x = [1, 2.0, 3]

print(x[-1])
