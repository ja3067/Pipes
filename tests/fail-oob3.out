RuntimeError: list index out of bounds
